### ISSUE TITLE
Option to add mimetype to wps request

### ISSUE THREAD

**Issue by eran-pinhas** [geopython/OWSLib#539]
Hey guys

https://github.com/geopython/OWSLib/blob/916003071191b9c0a94a8d9a352bd87e42ab0e43/owslib/wps.py#L708-L711

in these lines the WPS creates the xml element that we will send as WPS execute payload as the output details.

Will it be possible to declare the mime type of the output? I thought of something like this:

```python
 elif isinstance(output, list): 
     for ouputTuple in output: 
        if(len(tuple) == 2):
          (identifier, as_reference) = tuple
        else:
          (identifier, as_reference, mimeType) = tuple
         # of course adjusting _add_output to handle the mimeType will be required
         self._add_output( 
             responseDocumentElement, identifier, asReference=as_reference, mimeType=mimeType) 
```
*In this way the backward compatibility

any opinions?

**Comment by eran-pinhas** [comment]
@cehbrecht

**Comment by cehbrecht** [comment]
fixed by PR #540.
